src/content/community.md

```diff
@@ -52,6 +53,39 @@ There are several ways to stay up-to-date with Firefox add-on technology and con
 
 Stay up to date with the latest information about add-on technology on the [Add-ons blog](https://blog.mozilla.org/addons).
 
+### Office hours <span id="office-hours"></span>
+
+The Firefox Add-ons team holds one-on-one office hours sessions to help developers make their extensions available on Android. Developers can also use the [Firefox for Android Add-ons](https://discourse.mozilla.org/c/add-ons/android/9393) forum to discuss issues asynchronously.
+
+<!-- Google Calendar Appointment Scheduling begin -->
+<link href="https://calendar.google.com/calendar/scheduling-button-script.css" rel="stylesheet">
+<script src="https://calendar.google.com/calendar/scheduling-button-script.js" async></script>
+<script>
+(function() {
+  var target = document.currentScript;
+  window.addEventListener('load', function() {
+    calendar.schedulingButton.load({
+      url: 'https://calendar.google.com/calendar/appointments/schedules/AcZssZ3yNnoZNBdJCTiJjqziXs9GoCYpfW9XZFkB0QWKNy9dfoqqPlCw7UsYh48Ff0i3lZcA5YzygwrU?gv=true',
+      color: '#039BE5',
+      label: 'Book an appointment',
+      target,
+    });
+  });
+})();
+</script>
+<!-- end Google Calendar Appointment Scheduling -->
+<style>
+/*
+ * Google's scripts inject a button styled with "Roboto" but doesn't load that
+ * font. Unfortunately, that means the font-family falls back to Times in
+ * Firefox and Chrome. Since we can't set an inline style on this element, the
+ * next best thing we can do is force the style on buttons in this this section.
+ */
+#connect-with-the-community button {
+  font-family: inherit !important;
+}
+</style>
```

Comment:
It would be good to move this to a scss file:

```diff
diff --git a/src/assets/css/_hacks.scss b/src/assets/css/_hacks.scss
new file mode 100644
index 0000000..423aca7
--- /dev/null
+++ b/src/assets/css/_hacks.scss
@@ -0,0 +1,9 @@
+/**
+ * Google's scripts inject a button styled with "Roboto" but doesn't load that
+ * font. Unfortunately, that means the font-family falls back to Times in
+ * Firefox and Chrome. Since we can't set an inline style on this element, the
+ * next best thing we can do is force the style on buttons in this this section.
+ */
+#connect-with-the-community button {
+  font-family: inherit;
+}
diff --git a/src/assets/css/styles.scss b/src/assets/css/styles.scss
index b3735ec..390d6bf 100644
--- a/src/assets/css/styles.scss
+++ b/src/assets/css/styles.scss
@@ -20,6 +20,7 @@ CSS file.
 @import 'contentguidelines';
 @import 'fullwidth';
 @import 'sidebar';
+@import 'hacks';
 
 // Select the components you want to use
 @include foundation-global-styles;
diff --git a/src/content/community.md b/src/content/community.md
index 5839c8a..3c99eae 100644
--- a/src/content/community.md
+++ b/src/content/community.md
@@ -66,7 +66,7 @@ The Firefox Add-ons team holds one-on-one office hours sessions to help develope
   window.addEventListener('load', function() {
     calendar.schedulingButton.load({
       url: 'https://calendar.google.com/calendar/appointments/schedules/AcZssZ3yNnoZNBdJCTiJjqziXs9GoCYpfW9XZFkB0QWKNy9dfoqqPlCw7UsYh48Ff0i3lZcA5YzygwrU?gv=true',
-      color: '#039BE5',
+      color: '#015fdf',
       label: 'Book an appointment',
       target,
     });
@@ -74,17 +74,6 @@ The Firefox Add-ons team holds one-on-one office hours sessions to help develope
 })();
 </script>
 <!-- end Google Calendar Appointment Scheduling -->
-<style>
-/*
- * Google's scripts inject a button styled with "Roboto" but doesn't load that
- * font. Unfortunately, that means the font-family falls back to Times in
- * Firefox and Chrome. Since we can't set an inline style on this element, the
- * next best thing we can do is force the style on buttons in this this section.
- */
-#connect-with-the-community button {
-  font-family: inherit !important;
-}
-</style>
 
 ### Questions about add-on development
 
```

Also the color of the button should probably be like the other one at the top.

Reply:
These suggestions have been integrated.